Gemfile

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+ruby '~> 3.1.1'
```

Comment:
Will this cause the app to break when https://github.com/sul-dlss/puppet/pull/7883 is released?

Reply:
It's one of the things I'm looking to test.